### PR TITLE
Support bbqr psbts

### PR DESCRIPTION
**Pull request by Kukks**
https://bbqr.org/ @nvk

### CODE REVIEW

**Comment by NicolasDorier** [comment]
Need test in `CheckJSContent` which check this script hasn't be tempered with.

**Comment by Kukks** [comment]
> Need test in `CheckJSContent` which check this script hasn't be tempered with.

done

File: BTCPayServer/Views/Shared/CameraScanner.cshtml [hunk 1]
```diff
@@ -146,12 +154,51 @@ function initCameraScanningApp(title, onDataSubmit, modalId, submitOnScan = fals
 			onDecode(content) {
 				if (this.qrData) return;
                 const isUR = content.toLowerCase().startsWith("ur:");
-                console.debug(1, content);
-                
+                const isBBQr = content.startsWith("B$");
+                console.debug(content);
```
Comment (dennisreimann) [inline BTCPayServer/Views/Shared/CameraScanner.cshtml:158]:
```suggestion
```

**Review by dennisreimann — COMMENTED** [review]
LGTM

File: BTCPayServer/Views/Shared/CameraScanner.cshtml (hunk 1 above)
Comment (Kukks) [reply BTCPayServer/Views/Shared/CameraScanner.cshtml:158]: let's leave it, was there before and will be useful when debugging something that changed

**Comment by nvk** [comment]
@Kukks is this deployed?

**Comment by pavlenex** [comment]
@nvk It is! Part of 1.13.0 major release feature set. 
 https://blog.btcpayserver.org/btcpay-server-1-13-0/#-better-bitcoin-qr-support

**Pull request merged**